ckan/lib/helpers.py

```diff
@@ -383,6 +384,14 @@ def url_for(*args, **kw):
     return _local_url(my_url, locale=locale, **kw)
 
 
+def _coerce_for_querystring(value):
+    if value is None:
+        return 'null'
+    if isinstance(value, numbers.Number):
```

Comment:
note we don't need to explicitly check for boolean here because bool is a subclass of int

```
>>> import numbers
>>> isinstance(True, numbers.Number)
True
>>> isinstance(False, numbers.Number)
True
```